src/keys.rs

```diff
@@ -133,11 +137,25 @@ impl PartialEq for SpendValidatingKey {
     }
 }
 
+impl Eq for SpendValidatingKey {}
+
 impl SpendValidatingKey {
     /// Randomizes this spend validating key with the given `randomizer`.
     pub fn randomize(&self, randomizer: &pallas::Scalar) -> redpallas::VerificationKey<SpendAuth> {
         self.0.randomize(randomizer)
     }
+
+    /// Converts this spend validating key to its serialized form.
+    pub(crate) fn to_bytes(&self) -> [u8; 32] {
+        <[u8; 32]>::from(&self.0)
```

Comment:
```suggestion
        // This is correct because the wrapped point must have ỹ = 0, and
        // so the point repr is the same as I2LEOSP of its x-coordinate.
        <[u8; 32]>::from(&self.0)
```